generators/newapp/templates/dot-dockerignore.tmpl

```diff
@@ -0,0 +1,3 @@
+node_modules/
+*.log
+bin/
```

Comment:
I originally had that, but then the when building the binary we no longer have the git information needed to build the `ldflags`, so the SHA doesn't get burnt into the binary. People can add `.git`, and anything else they want, this is meant to just be a "starter" file.